docs/pages/_app.js

```diff
@@ -43,12 +53,9 @@ ponyfillGlobal.muiDocConfig = {
   csbIncludePeerDependencies: (deps, { versions }) => {
     const newDeps = { ...deps };
 
-    if (
-      newDeps['@material-ui/x'] ||
-      newDeps['@mui/x-data-grid-pro'] ||
-      newDeps['@mui/x-data-grid']
-    ) {
+    if (newDeps['@mui/x-data-grid-pro'] || newDeps['@mui/x-data-grid']) {
       newDeps['@material-ui/core'] = versions['@material-ui/core'];
+      newDeps['@material-ui/styles'] = versions['@material-ui/styles'];
```

Comment:
Should we switch to `@mui` for core repo on this file since we installed them ?

https://github.com/mui-org/material-ui-x/pull/2441/files#diff-adfa337ce44dc2902621da20152a048dac41878cf3716dfc4cc56d03aa212a56R28